deptry/utils.py

```diff
@@ -39,3 +41,64 @@ def load_pyproject_toml(pyproject_toml_path: str = PYPROJECT_TOML_PATH) -> Dict[
             return tomllib.load(pyproject_file)
     except FileNotFoundError:
         raise FileNotFoundError(f"No file `pyproject.toml` found in directory {os.getcwd()}")
+
+
+def find_site_packages_below(root: Path, platform: str) -> Optional[Path]:
+    """Find site packages directory below a virtual environment root"""
+    if platform == "Windows":
+        pattern = "Lib/site-packages"
```

Comment:
Would it be an idea to just look for `site-packages` recursively in this directory? Then we can keep the code platform agnostic

Reply:
Definitely! I thought however that being as specific as possible would reduce the risk of errors since we are doing quite a guesswork here 😅 . But I updated with a compromise: check all the specific cases!

Reply:
Your new proposal looks good to me :)

Reply:
Implemented in [3867c1d](https://github.com/fpgmaas/deptry/pull/207/commits/3867c1dd45db69c72ca768b5fbceabe27733afe2), and when `running_in_virtualenv` was left with a single `ctx` argument, I thought it might as well be calle `self` 😅